scripts/install/installnode.py

```diff
@@ -0,0 +1,463 @@
+from JumpScale import j
+import requests
+import os
+
+class Node():
+    """ Install basic packages on a node """
+    def __init__(self, config):
+        self.config = config
+
+    def prepare_config(self):
+        self.nginx = {
+            'servicename': 'nginx',
+            'instance': 'system',
+            'daemonpath': './nginx ',
+            'args': '-c /opt/nginx/cfg/nginx.conf',
```

Comment:
we decided to use the system nginx which should already be registered just need to reload it cause we add files under /etc/nginx/cfg/sites-enabled (from the image)